### DISCUSSION TITLE
Error - The server does not support SSL connections [Jul 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2878] [Discord Threads]
> 🙋‍♂️ **lublak**  
#### Is it possible to point nocodb to the database and show all schemas in nocodb?  
[message.txt](https://cdn.discordapp.com/attachments/661905455894888493/999286572006514789/message.txt)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@100849859006840832> Could you help us with more details about your project (using docker/npm/npx) & where your database is hosted?   
####    
#### this looks to be a database configuration error. confirm if your db configurations are right by connecting via db viewer like tablePlus/pgAdmin  
#  
> 🙋‍♂️ **lublak**  
#### <@857998597474484295> my database is hosted localhost. i used npx setup which creates a .env file for me.   
#### It looks something like this:   
#### `NC_DB=pg://localhost:5432?u=postgres&p=password&d=database`  
#  
> 🙋‍♂️ **lublak**  
#### I i connect manually with localhost all works fine  
#  
> 🙋‍♂️ **lublak**  
#### <@857998597474484295> i can remove my .env file. If i than run npm run start i can use: create project with external database  
#  
> 👨‍🦲 **dstala**  
#### ok. will check if I can reproduce issue locally with npx & .env file.  
#  
> 🙋‍♂️ **lublak**  
#### <@857998597474484295> do you know what the problem is? Currently I can simply use sqlite3 and create projects as external data sources.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999286572274962562)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
